api/envoy/api/v2/endpoint/load_report.proto

```diff
@@ -19,10 +20,47 @@ message UpstreamLocalityStats {
   // collected from. Zone and region names could be empty if unknown.
   core.Locality locality = 1;
 
-  // The total number of requests sent by this Envoy since the last report. A
+  // The total number of requests sent by this Envoy since the last report. This
+  // information is aggregated over all the upstream Endpoints. total_requests
+  // can be inferred from:
+  //
+  // .. code-block:: none
+  //
+  //   total_requests = total_successful_requests + total_requests_in_progress +
+  //     total_error_requests
+  //
+  // The total number of requests successfully completed by the endpoints in the
+  // locality.
+  uint64 total_successful_requests = 2;
```

Comment:
Where did we end up with on the "let's eventually deprecate cluster granularity stats and have the management server aggregate the endpoint stats"?

Reply:
I just caught up on this, ignore this comment.

Reply:
I thought we did not want to disable a scenario where Server (Balancer) might not need per endpoint granurality information. In which case we can avoid sending all the Endpoint information to the Server.

To make this work I have added the following in LoadStatsResponse below

>  bool report_endpoint_granularity = 3;

Reply:
FWIW I think it makes sense to make this be configurable. If users don't want all the extra data there seems no reason to spend resources serializing it.